internal/venice-client-common/src/main/java/com/linkedin/venice/utils/ArrayUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.linkedin.venice.utils;
+
+public final class ArrayUtils {
+  private ArrayUtils() {
+  }
+
+  /**
+   * This is mostly a copy of Arrays.compareUnsigned(T[], T[]) from Java 9.
+   */
+  public static int compareUnsigned(byte[] a, byte[] b) {
+    if (a == b)
```

Comment:
though this is copy IMO we should still follow Venice codebase style

Reply:
Oh, yeah. Totally agree. Will fix it